tests/test_flows.sh

```diff
@@ -84,6 +84,19 @@ send_CAPI_command() {
     capi_command_reply=$(tr  -d '\r' < "$capi_command_result" | awk 'NR==2')
 }
 
+send_CAPI_1905() {
+    container="$1"; shift
+    dest="$1"; shift
+    messagetype="$1"; shift
+    command="DEV_SEND_1905,DestALid,${dest},MessageTypeValue,${messagetype}"
+    for arg; do
+        command="$command,$arg"
+    done
+    capi_command_mid=""
```

Comment:
Why is this needed? you set this 2 lines below

Reply:
(For future reference) It is needed because of the `return $?` in the following line, to make sure the mid is not set if the command fails.